openadapt/app/dashboard/app/settings/record_and_replay/form.tsx

```diff
@@ -32,19 +32,19 @@ export const Form = ({
         <form onSubmit={form.onSubmit(saveSettings(form))}>
             <Grid>
                 <Grid.Col span={6}>
-                    <Checkbox label="Record window data" {...form.getInputProps('RECORD_WINDOW_DATA')} checked={form.values.RECORD_WINDOW_DATA} />
```

Comment:
Won't this prevent the checkboxes from being checked appropriately depending on the configuration values?